types/interaction.d.ts

```diff
@@ -2,6 +2,8 @@ export interface IPointer {
     target: number;
     x: number;
     y: number;
+    targetX?: number;
```

Comment:
>targetX [](start = 4, length = 7)

What are target X/Y vs x/y?

Reply:
Added a comment. It's relative (x,y) to the target element. We compute it for click events.